frontend/src/metabase/visualizations/components/ObjectDetail/ObjectDetail.styled.tsx

```diff
@@ -54,9 +54,6 @@ export const CloseButton = styled.div`
   margin-left: 1rem;
   padding-left: 1rem;
   border-left: 1px solid ${color("border")};
-  ${breakpointMinMedium} {
```

Comment:
So there's something weird going on here - when the button disappears from view, it causes the keyboard tab behavior in the modal to go haywire, bringing the tabbing behavior back to the navbar and everything that's behind the modal. When I take this breakpoint out, the focus trap works again, but obviously we don't want to have the button that don't correspond to the breakpoint.

Reply:
I changed `display: none` to this weird absolute positioning thing I found as an alternative, but it's disgusting